README.md

```diff
@@ -1,11 +1,11 @@
-[![Visual Studio Team services](https://mseng.visualstudio.com/_apis/public/build/definitions/698eacea-9ea2-4eb8-80a4-d06170edf6bc/5908/badge)](https://mseng.visualstudio.com/vsts-cli/_build/index?context=mine&path=%5C&definitionId=6169&_a=completed)
+[![Azure DevOps Services](https://mseng.visualstudio.com/_apis/public/build/definitions/698eacea-9ea2-4eb8-80a4-d06170edf6bc/5908/badge)](https://mseng.visualstudio.com/vsts-cli/_build/index?context=mine&path=%5C&definitionId=6169&_a=completed)
 [![Python](https://img.shields.io/pypi/pyversions/vsts-cli.svg)](https://pypi.python.org/pypi/vsts-cli)
 
-# Microsoft Visual Studio Team Services CLI 
+# Microsoft Azure DevOps Services CLI 
 
-VSTS CLI is a new command line interface for [Visual Studio Team Services](https://www.visualstudio.com/team-services/) (VSTS) and [Team Foundation Server](https://www.visualstudio.com/tfs/) (TFS) 2017 Update 2 and later.
+Azure DevOps/VSTS CLI is a new command line interface for [Azure DevOps Services](https://www.visualstudio.com/team-services/) (VSTS) and [Azure DevOps Server](https://www.visualstudio.com/tfs/) (TFS) 2017 Update 2 and later.
```

Comment:
I don't think we should conflate these two names. Are you making these updates to clarify the change in name?

Reply:
Yes. People might get confused, so referring to some places with both as Azure DevOps\VSTS CLI, I believe we could rename it to Azure DevOps CLI by omitting  VSTS CLI.